.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+## Bug Description
+<!--
+Please provide a clear and concise description of what the bug is.
+
+What did you expect to happen? What actually happened?
+
+For bugs with fixers: How the code was fixed? How did you expect the code to be fixed?
+-->
+
+## Minimal Code Snippet
+<!-- Please provide example code that demonstrates the issue. Do NOT paste screenshots of code! -->
+
+```php
+// Place your code sample here.
+```
+
+## Error Code
+<!--
+The error code for the sniff that is (or should be) being triggered (you
+can see the sniff error codes by running `phpcs` with the `-s` flag).
+e.g. `WordPress.PHP.NoSilencedErrors.Discouraged`
+-->
```

Comment:
Maybe add: "_You can leave this section empty if you are reporting as false negative_" ?